### PR TITLE
doc/rbd: add clone mapping command

### PR DESCRIPTION
**Pull request by zdover23**
Add a command that explains how to map a formatted clone when the parent image and the formatted clone have different encryption types.

## Contribution Guidelines
- To sign and title your commits, please refer to [Submitting Patches to Ceph](https://github.com/ceph/ceph/blob/main/SubmittingPatches.rst).

- If you are submitting a fix for a stable branch (e.g. "quincy"), please refer to [Submitting Patches to Ceph - Backports](https://github.com/ceph/ceph/blob/master/SubmittingPatches-backports.rst) for the proper workflow.

- When filling out the below checklist, you may click boxes directly in the GitHub web UI.  When entering or editing the entire PR message in the GitHub web UI editor, you may also select a checklist item by adding an `x` between the brackets: `[x]`.  Spaces and capitalization matter when checking off items this way.

## Checklist
- Tracker (select at least one)
  - [ ] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [x] Doc update (no ticket needed)
  - [ ] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [x] No impact that needs to be tracked
- Documentation (select at least one)
  - [x] Updates relevant documentation
  - [ ] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [x] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`
- `jenkins test windows`
- `jenkins test rook e2e`
</details>

### CODE REVIEW

File: doc/rbd/rbd-encryption.rst [hunk 1]
```diff
@@ -252,6 +252,13 @@ Here is an example of a command that maps a formatted clone:
 
    rbd device map -t nbd -o encryption-passphrase-file=clone-passphrase.bin -o encryption-passphrase-file=passphrase.bin mypool/myclone
 
+If the parent image and the clone are formatted with different encryption
+formats, use a command of this form:
+
+.. prompt:: bash #
+
+   rbd device map -t nbd -o encryption-format=luks2,encryption-passphrase-file=clone-passphrase.bin,encryption-format=luks1,encryption-passphrase-file=passphrase.bin mypool/myclone
```
Comment (idryomov) [inline doc/rbd/rbd-encryption.rst:260]: I don't think this is needed. `encryption-format` option default works for both LUKS1 and LUKS2, so the shorter variant that specifies only `encryption-passphrase-file` options would work just fine.

Comment (idryomov) [reply doc/rbd/rbd-encryption.rst:260]: Let's get rid of this paragraph and just merge `encryption-passphrase-file` options in the above example under a single `-o` -- a one-line change ;)

Comment (zdover23) [reply doc/rbd/rbd-encryption.rst]: @idryomov: I think I've done it the way you've asked, but let me know if I need to change it.

**Review by idryomov — APPROVED** [review]

**Comment by zdover23** [comment]
https://github.com/ceph/ceph/pull/56207 - Squid backport
https://github.com/ceph/ceph/pull/56208 - Reef backport
https://github.com/ceph/ceph/pull/56209 - Quincy backport

**Pull request merged**